packages/cloud-config/yipfiles/01_defaults.yaml

```diff
@@ -1,25 +1,8 @@
 # Default cOS OEM configuration file
 stages:
-   initramfs.after:
-     - name: "Create needed folders"
-       commands:
-       - mkdir /etc
-     - name: "Mount volatile /etc"
-       commands:
-       - mount -t tmpfs -o size=20m tmpfs /etc
-     - name: "Write fstab"
-       files:
-        - path: /etc/fstab
-          content: |
-                    tmpfs   /etc         tmpfs   rw,nodev,nosuid,size=20MB          0  0
-#                    LABEL=COS_OEM        /oem       ext4  defaults                    0  1
-#                    LABEL=COS_PERSISTENT /usr/local ext4  data=ordered                0  2
-#                    LABEL=COS_GRUB       /boot/efi  vfat  defaults                    0  0
-          permissions: 0644
-          owner: 0
-          group: 0
+    fs.after:
```

Comment:
I changed to apply all on the fly settings over the ephemeral /etc. Then, since /etc is the default one from Tumbleweed untouched I just run few tweaks and the whole yaml can be reduced.